src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_errors.py

```diff
@@ -0,0 +1,125 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+class ErrorClass:
+    error_title = ""
+    error_message = ""
+
+    def __init__(self, title, message):
+        self.error_title = title
+        self.error_message = message
+
+    def get_error_message(self, additional_message=None):
+        if additional_message:
+            return "An error occurred: {}\n{}\n{}".format(self.error_title, self.error_message, additional_message)
+        else:
+            return "An error occurred: {}\n{}".format(self.error_title, self.error_message)
+
+    def set_error_message(self, message):
+        return ErrorClass(self.error_title, message)
+
+    def append_error_message(self, message):
+        return self.set_error_message(self.error_message + "\n" + message)
+
+    def format_error_message(self, *args):
+        return self.set_error_message(self.error_message.format(*args))
+
+
+# DOCKER ERRORS
+
+
```

Comment:
nit: remove extra 2 lines after comment from all places. Usually, comment should be right above the section you are referring to.

Reply:
I see. Removed